roles/addons/diskless/vars/RedHat.yml

```diff
@@ -1,2 +1,7 @@
 ---
 diskless_apache_user: apache
+diskless_packages_to_install: 
+  - httpd
+  - clustershell
```

Comment:
Is clustershell required or python3-clustershell is enough?

Reply:
python3-clustershell is enough. Fixed.